rhel/SPECS/dbaas-controller.spec

```diff
@@ -0,0 +1,67 @@
+%undefine _missing_build_ids_terminate_build
+%global _dwz_low_mem_die_limit 0
+
+# do not strip debug symbols
+%global debug_package     %{nil}
+
+%global provider          github
+%global provider_tld      com
+%global project           percona-platform
+%global repo              dbaas-controller
```

Comment:
is it ok that package name would be dbaas-controller and not percona-dbaas-controller?

Reply:
Yeah, that's similar to pmm-managed, etc